addons/account_edi_ubl_cii/wizard/account_move_send.py

```diff
@@ -84,14 +84,17 @@ def _compute_ubl_warnings(self):
             not_configured_partners = wizard.move_ids.partner_id.commercial_partner_id.filtered(
                 lambda partner: not (partner.peppol_eas and partner.peppol_endpoint)
             )
-            if len(not_configured_partners) == 1:
-                wizard.ubl_partner_warning = _("This partner is missing Peppol EAS or Peppol Endpoint field. "
-                                        "Please check those in its Accounting tab or the generated file will be incomplete.")
-            if len(not_configured_partners) > 1:
-                names = ', '.join(not_configured_partners[:5].mapped('display_name'))
-                wizard.ubl_partner_warning = _("The following partners are missing Peppol EAS or Peppol Endpoint field: %s. "
-                                        "Please check those in their Accounting tab. "
-                                        "Otherwise, the generated files will be incomplete.", names)
+            if not_configured_partners:
+                wizard.ubl_partner_warning = {
+                    'ubl_partner_warning': {
+                        'message': _("These partners are missing Peppol EAS or Peppol Endpoint field. "
+                                    "Please check those in their Accounting tab. "
+                                    "Otherwise, the generated files will be incomplete."),
```

Comment:
```suggestion
                                     "Please check those in their Accounting tab. "
                                     "Otherwise, the generated files will be incomplete."),
```
These lines are under indented :sweat_smile: